cfgov/jinja2/v1/newsroom/press-resources/index.html

```diff
@@ -127,7 +127,7 @@ <h3 class="contact-person_name">Sam Gilford</h3>
             </section>
 
             <section class="contact-person content-l_col-1-2">
-                <h3 class="contact-person_name">David Mayorga</h3>
+                <h3 class="h4">David Mayorga</h3>
```

Comment:
It's okay that these names are no longer reduced in size at mobile? https://github.com/cfpb/cfgov-refresh/blob/content-updates/cfgov/unprocessed/css/pages/press-resources.less#L66